cs-engine/1.12/index.md

```diff
@@ -45,7 +45,7 @@ to update its RHEL kernel.
     $ sudo rpm --import "https://sks-keyservers.net/pks/lookup?op=get&search=0xee6d536cf7dc86e2d7d56f59a178ac6c6238f52e"
     ```
 
-    > **Note**: If the key server above does not respond, you can try one of these:
+    > **Note:** If the key server above does not respond, you can try one of these:
```

Comment:
In the documentation style guide, the "Note" style example given includes the colon in the bold text as follows. 
![note](https://cloud.githubusercontent.com/assets/25146885/23779355/f6b15148-057a-11e7-9eb4-95cca545dbab.png)
So, would you please update the style guide? Thanks.